pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts

```diff
@@ -0,0 +1,55 @@
+import { ProviderVmware } from '../../../models/providerVmware';
+import {
+  clickByText,
+  createNamespace,
+  getStarted,
+  login,
+  provisionNetwork,
+} from '../../../../utils/utils';
+import { vmwareTier1TestCephCold } from './tier1_config_vmware';
+import {
+  vmwareNetworkMapping_2x_network,
+  vmwareNetworkMappingSingle,
+  vmwareProviderUser,
+} from './tier1_config_vmware';
+import { button, SEC } from '../../../types/constants';
+import { MappingNetwork } from '../../../models/mappingNetwork';
+// import { MappingStorage } from '../../../models/mappingStorage';
+
+describe('Tier1 tests for testing provider', () => {
+  const provider = new ProviderVmware();
+  const networkMapping = new MappingNetwork();
+  // const storageMapping = new MappingStorage();
+  before('Login to MTV', () => {
+    // Clearing all cookies in local storage if any
+    cy.clearLocalStorageSnapshot();
+    login(vmwareTier1TestCephCold.loginData);
+    getStarted();
+    provider.create(vmwareProviderUser);
+    // Saving local storage state after login
+    cy.saveLocalStorage();
+    createNamespace(vmwareTier1TestCephCold.planData.namespace);
+    provisionNetwork(vmwareTier1TestCephCold.planData.namespace);
+  });
+
+  beforeEach('Login to MTV', () => {
```

Comment:
You might want to change the description on this one to be closer to what the function does.